.github/workflows/clp-core-build-macos.yaml

```
@@ -58,13 +58,32 @@ jobs:
+        run: |-
+          brew install coreutils
+          brew install llvm@16
+
       - name: "Install dependencies"
         run: "./components/core/tools/scripts/lib_install/macos/install-all.sh"
 
       - run: "./tools/scripts/deps-download/init.sh"
         shell: "bash"
 
       - run: "task deps:core"
+        env: >-
```

actionlint (1.7.4)

property "os" is not defined in object type {runner: string; use_shared_libs: bool}

(expression)